bee-node/src/plugins/version_checker.rs

```diff
@@ -40,3 +50,25 @@ impl<N: Node> Worker<N> for VersionChecker {
         Ok(Self::default())
     }
 }
+
+async fn is_latest(client: &Client) -> Result<bool, reqwest::Error> {
+    let res = client
+        .get(BEE_GIT_API_LATEST)
+        .header(USER_AGENT, "curl")
```

Comment:
Maybe we can put the user agent in the client builder so that we don't have to repeatedly set it ?
See https://docs.rs/reqwest/0.11.4/reqwest/struct.ClientBuilder.html#method.user_agent